fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/LuceneIndexTest.java

```diff
@@ -1153,7 +1144,6 @@ void testAutoCompleteSearchForPhrase() throws Exception {
                             "states united as a country",
                             "states have been united as a country",
                             "all the states united as a country",
-                            "all the states have been united as a country",
```

Comment:
This text is from a document that is then updated with a new document. The old behavior was to return both versions of the text, whereas the new behavior only retrieves the text from the current version of the document.

Reply:
I am not sure I am getting why this test data is not needed anymore.

Reply:
@alecgrieser sorry that was a mistake. My original purpose for that was not to use the same record ID. That text was supposed to be a separate record to verify the phrase search for auto-complete. Could you please revert this change and change its recordID to ```1631L```?

Reply:
Sure thing. Made that change